implementations/rust/ockam/ockam/src/profile.rs

```diff
@@ -1,22 +1,28 @@
-use crate::{Contact, OckamError};
+use crate::OckamError;
 use hashbrown::HashMap;
```

Comment:
switch to using ockam_core::lib::HashMap. That will use the correct one depending on no_std, alloc, or std

Reply:
Fixed